connect/src/com/telenor/connect/ui/ConnectLoginButton.java

```diff
@@ -122,6 +122,9 @@ public ConnectLoginButton(Context context, AttributeSet attrs, int defStyleAttr,
     @Override
     protected void onVisibilityChanged(@NonNull View changedView, int visibility) {
         super.onVisibilityChanged(changedView, visibility);
+        if (loginButton == null || loginButton.getActivity() == null) {
+            return;
```

Comment:
Should we log this as an error?

Reply:
No, because it's not an error. It's more about specific Android version behaviour.